docs/source/command_line.rst

```diff
@@ -449,6 +449,22 @@ in error messages.
 
         main.py:12:9: error: Unsupported operand types for / ("int" and "str")
 
+``--no-color-output``
+    This flag will disable color output in error messages, enabled by default.
+
+``--error-summary``
```

Comment:
I would document `--no-error-summary` (as you did above), since it is also enabled by default.